### PR TITLE
fix(ui): show system badge for system tags

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes :

Closes: #10487 

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :

Screenshot 2023-04-03 at 6 30 53 PM

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/common/Badge/Badge.style.less
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/common/Badge/Badge.style.less:1]: badge.style.less

**Review by ShaileshParmar11 — APPROVED** [review]
one miner comment, other than that looks good

**Review by ShaileshParmar11 — APPROVED** [review]

**Comment by harshach** [comment]
System badge icon needs to better. Putting the label doesn't look right to me and especially the color we are using.
Thoughts @sureshms

**Comment by sureshms** [comment]
@chirag-madlani and @harshach how about adding the system label only on the right side along with the glossary and glossary term details instead of the left panel?

**Comment by chirag-madlani** [comment]
@harshach @sureshms @devyani-kaushik 
Have added badge to details page, let me know your feedbacks if any

**Comment by chirag-madlani** [comment]
@harshach updated badge with new styles, screenshot updated.
P.S. I have kept Badge bit top aligned (superscripted) as discussed

**Review by harshach — APPROVED** [review]

**Pull request merged**